pkg/dbutil/common.go

```diff
@@ -756,3 +756,22 @@ func ignoreDDLError(err error) bool {
 		return false
 	}
 }
+
+// DeleteRows delete rows in several times. Only can delete less than 300,000 one time in TiDB.
+func DeleteRows(ctx context.Context, db *sql.DB, schemaName string, tableName string, where string, args []interface{}) error {
+	deleteSQL := fmt.Sprintf("DELETE FROM `%s`.`%s` WHERE %s limit 100000;", schemaName, tableName, where)
+	_, err := db.ExecContext(ctx, deleteSQL, args...)
```

Comment:
can we just check RowsAffected instead of `GetRowCount`

Reply:
https://github.com/pingcap/tidb-tools/pull/258/commits/bee3bb51f92908f70d171c6520124cdb5d2ab4db